src/index.html

```diff
@@ -6,22 +6,328 @@
+        <img
+          src="/src/images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list_element">
+            <a
+              href="#apple"
+              class="header__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#samsung"
+              class="header__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#smartphones"
+              class="header__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#laptop&computers"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#gadgets"
+              class="header__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#tablets"
+              class="header__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#photo"
+              class="header__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#video"
+              class="header__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Product image"
+        />
+        <h1 class="card__header">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__product_code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-counter">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </button>
+      </div>
+      <div class="card">
```

Comment:
[checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules): Add empty lines between multiline sibling blocks of HTML
remember to apply it to all cases in your project ;)